### PR TITLE
[iOS] Add a table header for showing the session start day

### PR DESCRIPTION
**Pull request by woxtu**
## Issue
- None

## Overview (Required)
- Just as the title

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by panpanini** [comment]
👀

File: frontend/ios/DroidKaigi 2019/Scenes/Session/SessionsViewModel.swift [hunk 1]
```diff
@@ -53,16 +55,21 @@ extension SessionsViewModel {
...
-                            self._error.accept(error.localizedDescription)
-                            return Observable.empty()
-                        }
-                }
+            .flatMap { [weak self] (_) -> Observable<SessionContents> in
+                guard let `self` = self else { return Observable.empty() }
+                return self.sessionRepository.fetch()
+                    .asObservable()
+                    .catchError { error in
+                        self._error.accept(error.localizedDescription)
```
Comment (panpanini) [inline frontend/ios/DroidKaigi 2019/Scenes/Session/SessionsViewModel.swift]: 🤔 do we need `weak self` here too? (I'm not too sure about iOS development...)

File: frontend/ios/DroidKaigi 2019/Scenes/Session/SessionTableViewCell.swift [hunk 2]
```diff
@@ -44,11 +44,11 @@ class SessionTableViewCell: UITableViewCell, Reusable {
         }
     }
 
-    var favoriteButtonDidTapped: ControlEvent<Void> {
+    var favoriteButtonDidTap: ControlEvent<Void> {
         return favoriteButton.rx.tap
     }
 
-    var bag = DisposeBag()
+    private(set) var bag = DisposeBag()
```
Comment (panpanini) [inline frontend/ios/DroidKaigi 2019/Scenes/Session/SessionTableViewCell.swift]: 📝 cool!

File: frontend/ios/DroidKaigi 2019/Scenes/Session/SessionsViewModel.swift [hunk 3]
```diff
@@ -53,16 +55,21 @@ extension SessionsViewModel {
             .subscribe(onNext: { [weak self] in self?.favoriteRepository.toggle(sessionId: $0.id_) })
             .disposed(by: bag)
         
+        let startDayText = input.topVisibleSession
+            .map { $0.startDayText }
```
Comment (takahirom) [inline frontend/ios/DroidKaigi 2019/Scenes/Session/SessionsViewModel.swift]: You are using Kotlin Logic ✨

**Comment by woxtu** [comment]
🙌

**Pull request merged**